src/components/calcite-action-pad/calcite-action-pad.scss

```diff
@@ -20,12 +15,41 @@
   padding-top: 0;
 }
 
-::slotted(calcite-action-group:last-child) {
-  border-bottom: none;
+.container {
+  display: inline-flex;
+  flex-direction: column;
+  box-shadow: var(--calcite-app-shadow-2);
+  max-width: 15vw;
+  overflow-y: auto;
 }
 
 .action-group--bottom {
   padding-bottom: 0;
   flex-grow: 1;
   justify-content: flex-end;
 }
+
+:host([layout="horizontal"], [expanded]),
```

Comment:
@asangma is the `expanded` needed here? Seems like the styles apply whether expanded or not.

Reply:
Ha! I just removed that. :)

Reply:
Thanks! looks good